distributed/utils.py

```diff
@@ -1331,6 +1358,9 @@ def cli_keywords(d: dict, cls=None):
         The keywords to convert
     cls: callable
         The callable that consumes these terms to check them for validity
+    cmd: string or object
+        A string with the name of a module, or the module containing a
+        click-generated command with a "main" function, or the function itself
```

Comment:
This explains that the `cmd` option takes but not what it is for. Perhaps some reference that this is the python module to run to start the worker would be helpful.